src/Libraries/RevitNodes/Elements/Element.cs

```diff
@@ -532,6 +532,19 @@ internal IEnumerable<Autodesk.Revit.DB.GeometryObject> InternalGeometry(bool use
                 }
             }
 
+            // FamilySymbol is a special case: the symbol must be activated before geometry is available, unless an instance is already present in the model
+            else if ((thisElement is FamilySymbol) && (geomElement == null || !geomElement.Any()))
+            {
+                var fs = (FamilySymbol)thisElement;
+                if (!fs.IsActive)
+                {
+                    fs.Activate();
+                    DocumentManager.Regenerate();
```

Comment:
The documentation at https://www.revitapidocs.com/2015/69706e35-87cc-a6d9-68fe-90a41c1c48db.htm does not mention that regeneration is required, but Scott C suggested it. Looking into whether this is strictly necessary or whether we need to update the Revit API docs.

Reply:
Scott C reminded me that that site is not official, the official docs do mention the regeneration requirement.

Reply:
There is a `DocumentManager.Regenerate()` at the beginning of this method, could we activate this `FamilySymbol` also at the beginning?

Reply:
Seems this will be one time action, I think it can just be here.